packages/react-components/react-skeleton/src/components/SkeletonLine/useSkeletonLine.ts

```diff
@@ -12,17 +12,18 @@ import type { SkeletonLineProps, SkeletonLineState } from './SkeletonLine.types'
  * @param ref - reference to root HTMLElement of SkeletonLine
  */
 export const useSkeletonLine_unstable = (props: SkeletonLineProps, ref: React.Ref<HTMLElement>): SkeletonLineState => {
+  const { height = '16px', width } = props;
+
+  const root = getNativeElementProps('div', {
+    ref,
+  });
```

Comment:
test fail is due to not having the props spread:

```suggestion
  const root = getNativeElementProps('div', {
    ref,
    ...props,
  });
```